app/utils/gadm.py

```diff
@@ -0,0 +1,37 @@
+GADM_41_IDS_MISSING_REVISION = (
+    "IDN.35.4",
+    "IDN.35.8",
+    "IDN.35.9",
+    "IDN.35.13",
+    "IDN.35.14",
+)
+
+
+def extract_level_id(adm_level: int, id_string: str):
```

Comment:
A note for next time we update GADM, if at all: we should have a script that normalizes all IDs and splits them into separate columns, instead of having to code around all these things

Reply:
I have a script which generates the normalized (de-accented, de-capitalized) fields, but didn't know about these anomalies at the time.

Reply:
I will modify it to take care of these anomalies, in case we have to import again anyway. We could even have it strip the _<revision>, and do away with the LIKE!